File: Ej2.py
import random
def aleatorios(n):
    pares=""
    impares=""
    for i in range(n):
        numero=random.randint(1,100)
        if(numero%2==0):
            pares=f"{pares} {numero},"
        else:
            impares=f"{impares} {numero},"
    return f"Pares: {pares}\nImpares: {impares}"
n=int(input())
print(aleatorios(n))

File: Ej2.ts
import readline from "node:readline/promises";
import {stdin as input, stdout as output} from "node:process";

const rl=readline.createInterface({input,output});
function aleatorios(n:number){
    let pares="",impares=""
    for(let i=0;i<n;i++){
        let numero=Math.ceil(Math.random()*100)
        if(numero%2==0)pares=`${pares} ${numero},`
        else impares=`${impares} ${numero},`
    }
    return `Pares ${pares}\nImpares ${impares}`
}
let n=Number(await rl.question(""));
console.log(aleatorios(n));
rl.close()